Sources/JWTKit/ECDSA/ECDSAError.swift

```diff
@@ -4,4 +4,5 @@ enum ECDSAError: Error {
     case signFailure
     case noPublicKey
     case noPrivateKey
+    case wrongCurve
```

Comment:
... Not to mention this is misnamed - it's not a "wrong" curve (as in "you picked the incorrect type", it's an unrecognized or invalid curve (as in "the curve you named isn't known").